.github/workflows/pull_request_build.yml

```diff
@@ -5,7 +5,8 @@ on:
 
 env:
   mops_version: 0.12.1
-  moc_version: 0.9.0
+  moc_version: 0.10.2
```

Comment:
We should also update the mops version. I think we can set it to "latest" because mops is unlikely to be critical for success of the run.